tests/layers/test_nms_rotated.py

```diff
@@ -87,6 +88,13 @@ def test_batched_nms_rotated_0_degree_cpu(self):
             backup = boxes.clone()
             keep_ref = batched_nms(boxes, scores, idxs, iou)
             assert torch.allclose(boxes, backup), "boxes modified by batched_nms"
+            # test batched_nms's scriptability
+            if TORCH_VERSION >= (1, 7):
```

Comment:
does it need 1.7? 
If it's about `unique`, it should be supported in 1.6: https://github.com/pytorch/pytorch/commit/eb3e9872c9ed038ead67a832e7e5295628cbb55a

Reply:
let me test it in 1.6